pkgs/top-level/all-packages.nix

```diff
@@ -37554,6 +37554,8 @@ with pkgs;
 
   enlightenment = recurseIntoAttrs (callPackage ../desktops/enlightenment { });
 
+  expidus = callPackage ../desktops/expidus {};
```

Comment:
I think we need a recurseIntoAttrs or similar here otherwise ofborg is not building the package

Reply:
Done